.travis.yml

```diff
@@ -103,17 +105,28 @@ install:
     elif [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
       MINCONDA_OS=$MINCONDA_OSX;
     fi
-  - wget "http://repo.continuum.io/miniconda/Miniconda3-$MINCONDA_VERSION-$MINCONDA_OS.sh" -O miniconda.sh;
-  - bash miniconda.sh -b -p $HOME/miniconda
-  - export PATH="$HOME/miniconda/bin:${PATH}"
-  - export INCLUDE_PATH="$HOME/miniconda/include:${INCLUDE_PATH}"
-  - hash -r
+
+  - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then
+      export CMAKE_GENERATOR="Visual Studio 15 2017 Win64" ;
+      export TESTS_TARGET="RUN_TESTS";
+      choco install --yes miniconda3 ;
+      source C:/Tools/miniconda3/Scripts/activate ;
+    else
+      export CMAKE_GENERATOR="Ninja" ;
+      export TESTS_TARGET="test";
+      wget "http://repo.continuum.io/miniconda/Miniconda3-$MINCONDA_VERSION-$MINCONDA_OS.sh" -O miniconda.sh;
+      bash miniconda.sh -b -p $HOME/miniconda ;
+      source $HOME/miniconda/bin/activate;
+      hash -r ;
+    fi
   - conda config --set always_yes yes --set changeps1 no
   - conda update -q conda
-  - conda install cmake -c conda-forge
-  - conda install xtl -c conda-forge
-  - conda install xsimd -c conda-forge
-  - conda install xtensor -c conda-forge
+  - conda install -c conda-forge xtl xsimd xtensor
+  - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then
+      conda install -c conda-forge boost hdf5 pkgconfig;
+      conda install -c ccordoba12 eigen3;
+      export PKG_CONFIG_PATH=C:/Tools/miniconda3/pkgs/eigen3-3.2.5-1/Library/lib/pkgconfig;
```

Comment:
This will fail right after Eigen gets a new version. Can we use find?

Reply:
Of course, sorry for that.